### PR TITLE
Removed unneeded RPC calls

### PR DESCRIPTION
**Pull request by rlaferla**
Removed unneeded RPC calls for getMsaId() and getLatestSchemaId() since state queries can be used instead.

Closes: #456

### CODE REVIEW

File: js/api-augment/README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Caches can also wreck this even if you reorder, so watch out.
 Optional responses are not mapped to `null` and instead return an object with a few properties.
 For more details see the [code for the Option class](https://github.com/polkadot-js/api/blob/master/packages/types-codec/src/base/Option.ts).
 ```javascript
-const optionalExample = await api.rpc.msa.getMsaId(account);
+const optionalExample = await api.rpc.schemas.getBySchemaId(1);
```
Comment (wilwade) [inline js/api-augment/README.md:86]: Good catch!

File: js/api-augment/interfaces/augment-api-query.ts [hunk 2]
```diff
@@ -292,6 +292,12 @@ declare module '@polkadot/api-base/types/storage' {
       [key: string]: QueryableStorageEntry<ApiType>;
     };
     msa: {
+     /**
+       * Storage type for key to MSA information
+       * - Key: AccountId
+       * - Value: [`MessageSourceId`]
+       **/
+      getMsaId: AugmentedQuery<ApiType, (arg: AccountId32 | string | Uint8Array) => Observable<Option<u64>>, [AccountId32]> & QueryableStorageEntry<ApiType, [AccountId32]>;
```
Comment (wilwade) [inline js/api-augment/interfaces/augment-api-query.ts:300]: Where is this coming from?

**Review by wilwade — COMMENTED** [review]
One question, otherwise looks good

File: js/api-augment/interfaces/augment-api-query.ts (hunk 2 above)
Comment (saraswatpuneet) [reply js/api-augment/interfaces/augment-api-query.ts:300]: I think this is a public function

Comment (saraswatpuneet) [reply js/api-augment/interfaces/augment-api-query.ts:300]: Yes this should not have been added

Comment (saraswatpuneet) [reply js/api-augment/interfaces/augment-api-query.ts:300]: @rlaferla  I think you also have to update metadeta fetch first and then build with chain running in instant sealing mode

**Review by saraswatpuneet — COMMENTED** [review]
Need following changes

1. Update runtime to remove the rpc from runtime too
2. Try updating js/api-augment again

File: js/api-augment/interfaces/augment-api-query.ts (hunk 2 above)
Comment (rlaferla) [reply js/api-augment/interfaces/augment-api-query.ts:300]: That's some stray code.  It's removed in the next commit.

**Review by enddynayn — APPROVED** [review]

**Review by saraswatpuneet — APPROVED** [review]

**Pull request merged**